extkeys/mnemonic.go

```diff
@@ -26,7 +26,7 @@ import (
 // https://github.com/bitpay/bitcore-mnemonic/blob/master/lib/mnemonic.js (used in eth-lightwallet.js)
 
 // Salt is input salt to mnemonic key
-const Salt = "status-im"
+const Salt = "mnemonic"
```

Comment:
does it need to be exported?

Reply:
oops, you beat me

Reply:
it was exported so I left it like this but I agree it's better to change it!